fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -2,6 +2,12 @@ module Fastlane
   class EnvironmentPrinter
     def self.output
       env_info = get
+
+      # Remove sensitive option values
+      FastlaneCore::Configuration.sensitive_strings.compact.each do |sensitive_element|
+        env_info.gsub!(sensitive_element, "#########")
```

Comment:
I don't think it matters to be honest, we could also put `"<REDACTED>"` in there to make it more clear that the value was automatically replaced.